arangod/Cluster/EnsureIndex.cpp

```diff
@@ -90,6 +90,12 @@ EnsureIndex::EnsureIndex(MaintenanceFeature& feature,
 
 EnsureIndex::~EnsureIndex() = default;
 
+// For local book keeping and reporting on /_admin/actions
+arangodb::Result EnsureIndex::setProgress(uint64_t u) {
```

Comment:
As progress is reported as a double value inside the index itself, wouldn't it be useful to make it a double value here as well?

Reply:
So, didn't change the base class, where this is coming from. Historic reasons.